### PR TITLE
(Content): Hai Sea Scorpion and Cicada Adjustments

### PR DESCRIPTION
**Pull request by EjoThims**
----------------------
**Content (Artwork / Missions / Jobs)**

+6 engine space for Cicada

Because this midsize transport was given the second lowest engine capacity out of all Hai ships (besting solely the Aphid), while having leftover space that literally no Hai engines can fill. 6 more lets it hold the next highest thruster (as an after market upgrade) while moving up only past the interceptor for total engine space. This increase leaves it tied with the Light Freighter it seems designed as a counterpart to.

While the stock engine configs are the same between the two, the Scarab still has nearly double again the space for upgrades, as well as carrying so much less weight anyway (nearly double base weight -and- the extra cargo room weight). For comparison, after market upgrade on Cicada from "Benga" thruster to "Biroo" takes it from 285 top speed and 137 accel to 502 and 235, but would leave the "Benga" steering at 55. Same switch on Scarab is from 353 and 254 to 622 and 425, but there's then still enough room to also upgrade the steering all the way to -"Bondir"-, going from 101 to 284.

-40 base mass and +20 weapon space for Sea Scorpion plus tweaks to the stock configuration for Sea Scorpion

Because in the same weight division, the Ladybug (an older, peace time design) actually beats out the Sea Scorpion in a broad range of categories in ways that I don't think the Sea Scorpion's benefits in turn quite account for.

Sea Scorpion does have 50 more outfit space, 3200 more shields (26% increase), 1300 more hull (36% increase), and that 1 extra gun port, but also it is 80 tons heavier dead empty (so 130 more if outfit space is full on both) and has 46 less cargo space, 5 more required crew (but only 3 more bunks), 400 less fuel capacity, 2 less weapon space, and 3 less engine space.

Ladybug -does- have higher drag and heat dissipation behind the scenes at least (5.4 and .85 vs Sea Scorpions 5.2 and .7, respectively), but it also has all that scanning and the small ramscoop as well. The higher drag is enough to keep it's top speed below the Sea Scorpion, but it still has better acceleration and turning both because of the large weight differences.

Dropping the base weight and increasing the weapon space of the Sea Scorpion seems like the best way to resolve this. Placing more of the end weight difference between the two on that higher outfit space means the Ladybug is still more maneuverable in the field but will have less of an edge there than at current, while the Sea Scorpion is that much faster at top speed as well as giving it more flexibility in using that increased outfit space for a broader arsenal on the more combat focused on the two ships.

Stock config for Sea Scorpion has also been adjusted to further accentuate price difference between these two (the main supposed drawback for the Ladybug, after all), primarily by downgrading the reactor (which isn't needed for the tracker focus anyway) in favor of more generalized equipment. This also helps subtly play up Unfettered resource shortages - they no longer are selling their biggest reactor or jammers honed against their brethren to random humans.

I thought it best to leave other variant adjustments (and any fleet replacements that result) for a separate PR to leave room for different adjustments here, as has seemed the norm. Including switch from @Hurleveur so this version isn't the one used in fleets, though. :)

### CODE REVIEW

**Review by Hurleveur — APPROVED** [review]
this makes the sea scorpion more distinct from the ladybug, and more warlike, whilst still keeping that mass difference big enough to be noticeable.

The change in layout is to make the ship more accessible early game, because you don't need a Boulder reactor to power 4 tracker pods, not if you're not fighting Hai at least, for sure. The Unfettered do not want you fighting any Hai, so a more flexible builds makes a lot of sense, taking the shipyard price down from 16M to 13M (and you can always modify it if you so choose).

File: data/hai/hai ships.txt [hunk 1]
```diff
@@ -805,6 +805,9 @@ ship "Pond Strider" "Pond Strider (Railgun)"
 
 
 
+# This default build is intended for the player,
+# whom the Unfettered do not want to arm properly for fighting the Hai (it lacks energy gen)
+# but instead with general plunder and utility in mind.
```
Comment (Zitchas) [inline data/hai/hai ships.txt:810]:
```suggestion
```

Stock/shipyard builds are supposed to be general functional builds, not optimized or specialized for a particular thing. As such, I don't think we need a comment explaining why this isn't an optimized build.

**Review by Zitchas — APPROVED** [review]
This makes sense.

File: data/hai/hai ships.txt (hunk 1 above)
Comment (Hurleveur) [reply data/hai/hai ships.txt:810]: yeah you're right, I just wanted to make sure nobody would use the default unfet variant against Hai and be like "its baaaad"
but that much should be obvious

Comment (Zitchas) [reply data/hai/hai ships.txt]: lol. If it's bad, and a regular opponent such that they have had experience, time, and opportunity to customize a loadout for that opponent (and the Unfettered clearly have against the Hai), then they should have their own configuration in a variant. Which already exists. :)

**Review by MasterOfGrey — APPROVED** [review]

**Pull request merged**